internal/controller/controllers/controller_event_wrapper_test.go

```diff
@@ -116,6 +117,48 @@ var _ = Describe("Controller events wrapper", func() {
 			Expect(numOfEvents(*cluster1.ID, nil)).Should(Equal(1))
 			Expect(numOfEvents(*cluster1.ID, host1.ID)).Should(Equal(1))
 		})
+
+		It("Sending a cluster event", func() {
```

Comment:
please add tests for `SendHostEventAtTime` and `SendClusterEventAtTime` as well.

Reply:
done